app/src/main/java/eu/kanade/tachiyomi/data/torrentServer/TorrentServerPreferences.kt

```diff
@@ -0,0 +1,39 @@
+package eu.kanade.tachiyomi.data.torrentServer
+
+import tachiyomi.core.preference.PreferenceStore
+
+class TorrentServerPreferences(
+    private val preferenceStore: PreferenceStore,
+) {
+    fun port() = preferenceStore.getString("pref_torrent_port", "8090")
+
+    fun trackers() = preferenceStore.getString(
```

Comment:
Why not use `getStringSet()`

Reply:
idk, luft added that

Reply:
@LuftVerbot

Reply:
If I change .getString to .getStringSet , what is this supposed to be? https://github.com/aniyomiorg/aniyomi/pull/1620/files#diff-3bbf39e312b060dd20c503cfa70fb4a29e6227dce19cdd4119991cc1affc35ccR429

Reply:
bruh I think it's better to leave it like that because adding the trackers one by one is an atrocity